sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/CombiningLimitsSuite.scala

```diff
@@ -111,7 +111,7 @@ class CombiningLimitsSuite extends PlanTest {
     comparePlans(optimized1, expected1)
 
     // test child max row > limit.
-    val query2 = testRelation.select().groupBy()(count(1)).limit(0).analyze
+    val query2 = testRelation2.select($"x").groupBy($"x")(count(1)).limit(1).analyze
```

Comment:
why do we need to modify the test case?

Reply:
The origin test case want to test child max row > limit. Because it uses limit value is 0, the new rules will eliminate it.
So I adjust the test case so that still test child max row > limit.